packages/terra-tabs/src/_TabMenu.jsx

```diff
@@ -115,6 +115,7 @@ class TabMenu extends React.Component {
         onKeyDown={this.handleOnKeyDown}
         className={cx(['tab-menu', { 'is-active': menuActive }])}
         data-terra-tabs-menu
+        id="tab-button"
```

Comment:
here too Id is not required. it has ref defined already.

Reply:
Removed in 15a749c